### DISCUSSION TITLE
Enable hex color code in color picker of the style customization panel

### DISCUSSION THREAD

**Discussion by hugosxm** [heyform/heyform#60] [Ideas]
Hi, 

When using the design customize panel : 

![heyform_colors](https://github.com/heyform/heyform/assets/23743455/7b82fe57-839c-4ddf-a012-e86a77b53f47)

I would like to be able to enter an hex code like #131313.

As a workarround I use the custom css box but it is less user friendly !

Thanks !

**Comment by iMuFeng** [comment]
Fixed in 2913579
